R/deployApp.R

```diff
@@ -377,16 +372,6 @@ deployApp <- function(
     )
   }
 
-  # Run checks prior to first saveDeployment() to avoid errors that will always
-  # prevent a successful upload from generating a partial deployment
-  if (!isCloudServer(accountDetails$server) && identical(upload, FALSE)) {
```

Comment:
Why did this block disappear? It ran against non-cloud servers.

Reply:
Oops, I think I missed the `!` here. I'll put this back.